asana.php

```diff
@@ -16,6 +16,7 @@
 define('ASANA_METHOD_PUT', 2);
 define('ASANA_METHOD_GET', 3);
 define('ASANA_METHOD_DELETE', 4);
+define('ASANA_RETURN_TYPE', 1); // 1- OBJECT | 2- ARRAY
```

Comment:
Better use this as a public property in the class instead of a file constant so it can be customized based on object instance.
You can define two constants to better set the property like `ASANA_RETURN_TYPE_OBJECT` and `ASANA_RETURN_TYPE_ARRAY` then you can do `public $returnType = ASANA_RETURN_TYPE_OBJECT;`